ajax/taskjob_logs.php

```diff
@@ -46,7 +46,6 @@
 }
 
 //unlock session since access checks have been done
-session_write_close();
```

Comment:
because, in PluginFusioninventoryTaskView::ajaxGetJobLogs i init a param in session if not set or provided by the taskjs